backend/app.js

```diff
@@ -1,46 +1,48 @@
 // Module imports
-import express from 'express'
-import path from 'path'
-import logger from 'morgan'
+import express from "express";
+import path from "path";
+import logger from "morgan";
 // Routes
-import { tokensRouter } from './routes/tokens.js'
-import { usersRouter } from './routes/users.js'
+import { tokensRouter } from "./routes/tokens.js";
+import { usersRouter } from "./routes/users.js";
 // import { postsRouter } from './routes/posts.js'
 // Config
-import './utils/envConfig.js'
-import './utils/databaseConnection.js'
+import "./utils/envConfig.js";
+import "./utils/databaseConnection.js";
 // Middleware
-import { catch404, errorHandler } from './utils/expresssMiddleware.js'
+import { catch404, errorHandler } from "./utils/expresssMiddleware.js";
 // Listeners
-import { normalizePort, onError, onListening } from './utils/serverListensers.js'
-import { petsRouter } from './routes/pets.js'
+import {
+  normalizePort,
+  onError,
+  onListening,
+} from "./utils/serverListensers.js";
 
-export const app = express()
+export const app = express();
 // setup for receiving JSON
-app.use(express.json())
-app.use(logger('dev'))
-app.use(express.static(path.join(process.cwd(), 'public')))
+app.use(express.json());
+app.use(logger("dev"));
+app.use(express.static(path.join(process.cwd(), "public")));
 // app.get('/', (req, res) => {
 //     res.send('Hello World!')
 // })
 
 // route setup
 // app.use('/posts', tokenChecker, postsRouter)
-app.use('/tokens', tokensRouter)
-app.use('/users', usersRouter)
-app.use('/pets', petsRouter)
```

Comment:
The pets router was the route where all the pets could be retrived from 1 endpoint.

Was there a reason it was deleted?

Reply:
`./routes/pets.js` doesn't exist on the branch main which this comes off.  When it's created (did you make and forget to commit?) we can add the route back in.  Currently the backend won't start because it's missing.